docs/amber_script.md

```diff
@@ -319,15 +317,15 @@ PIPELINE graphics kRedPipeline
 
   ATTACH kFragmentShader
 
-  FRAMEBUFFER kFrameBuffer
+  FRAMEBUFFER 256 256
   ENTRY_POINT kFragmentShader red
 END  # pipeline
 
 PIPELINE graphics kGreenPipeline
   ATTACH kVertexShader
   ATTACH kFragmentShader
 
-  FRAMEBUFFER kFrameBuffer
+  FRAMEBUFFER 256 256
```

Comment:
How do we encode the case where we want the framebuffer from the two pipelines to be the same, or to be different? My previous assumption was they'd have unique framebuffers, except in cases like this were I was passing the same FRAMEBUFFER into both of them.

Reply:
Example: 

       BIND myim AS fb DESCRIPTOR_SET 1 BINDING 0

Then the underlying resource (myim) is a declared buffer (image) that already exists.  And you can act on it twice, and it's orthogonal.

Reply:
ack